2d_PSF_code/Zernike_Module.py

```diff
@@ -7471,40 +7625,53 @@ def __init__(self, list_of_labelInput, list_of_spots, dataset,
         # folder containing the data taken with F/2.8 stop in April and May 2019
         # dataset 2
         if dataset == 2:
-            DATA_FOLDER = '/tigress/ncaplar/ReducedData/Data_May_28/'
+            #DATA_FOLDER = '/tigress/ncaplar/ReducedData/Data_May_28/'
+            #DATA_FOLDER = '/work/dev_2ddrp/ReducedData/'
+            pass
```

Comment:
At the moment you completely removed the ability to operate on the datasets that are not avaliable on Hilo. I would suggest keeping the possibility to work on this data on Tiger, and also writing here the folder destination where this data can be transferred at Hilo. This applies to all of the datasets.